memory_test.go

```diff
@@ -498,104 +498,104 @@ func TestMemArraysWithLenThreshold(t *testing.T) {
 		memLimit         uint64
 		expectedSizeDiff uint64
 	}{
-		{
-			desc: "array of numbers under length threshold",
-			script: `y = []
-			y.push([]);
-			let i=0;
-			checkMem();
-			for(i=0;i<20;i++){
-				y[0].push(i);
-			};
-			checkMem()`,
-			threshold: 100,
-			memLimit:  memUsageLimit,
-			expectedSizeDiff: SizeEmptyStruct + // Array overhead
-				20*SizeNumber, // size of property values
-		},
-		{
-			desc: "array of numbers over threshold",
-			script: `y = []
-			y.push([]);
-			let i=0;
-			checkMem();
-			for(i=0;i<200;i++){
-				y[0].push(i);
-			};
-			checkMem()`,
-			threshold: 100,
-			memLimit:  memUsageLimit,
-			expectedSizeDiff: SizeEmptyStruct + // Array overhead
-				200*SizeNumber, // size of property values
-		},
-		{
-			desc: "mixed array under threshold",
-			script: `y = []
-			y.push([]);
-			let i = 0;
-			checkMem();
-			for(i=0;i<100;i++){
-				y[0].push(i<50?0:true);
-			};
-			checkMem()`,
-			threshold: 200,
-			memLimit:  memUsageLimit,
-			expectedSizeDiff: SizeEmptyStruct + // Array overhead
-				(50 * SizeNumber) + (50 * SizeBool) + // (450) size of property values
-				// stack difference in going from
-				//	[..other, []]
-				//  to
-				//	[..other, true, 50]
-				+1,
-		},
+		// {
```

Comment:
[q] Should we add this back in or remove it?

Reply:
ahah I commented out by accident =/